gutenberg.php

```diff
@@ -181,12 +181,14 @@ function gutenberg_intercept_edit_post() {
 		return;
 	}
 
-	if ( isset( $_GET['post'] ) ) {
-		$post_ID = (int) $_GET['post'];
-		$post_id = $post_ID;
+	if ( empty( $_GET['post'] ) ) {
+		return;
 	}
 
-	if ( empty( $post_id ) ) {
+	$post_ID = (int) $_GET['post'];
+	$post_id = $post_ID;
```

Comment:
Why do we even do this?

Reply:
Ha, I also thought about what the situation might be. Couldn't think of anything but a hand-crafted invalid _edit post_ URL. 🤔 🤷‍♂️

Reply:
Just a bit silly to not assign this variable directly. Bit well...

Reply:
Oh, you mean why there are two variables...? Well, these are actually two different globals. See the import at the top of the function.

Reply:
Ah, I see. WordPress 🤷‍♀️